rollup/rollup_sync_service/rollup_sync_service.go

```diff
@@ -132,9 +143,12 @@ func (s *RollupSyncService) Start() {
 			case <-s.ctx.Done():
 				return
 			case <-syncTicker.C:
-				s.fetchRollupEvents()
+				err := s.fetchRollupEvents()
+				if err != nil {
+					log.Error("failed to fetch rollup events", "err", err)
+				}
 			case <-logTicker.C:
-				log.Info("Sync rollup events progress update", "latestProcessedBlock", s.latestProcessedBlock)
+				log.Info("Sync rollup events progress update", "latestProcessedBlock", s.callDataBlobSource.L1Height())
```

Comment:
```suggestion
				log.Info("Sync rollup events progress update", "latest processed block", s.callDataBlobSource.L1Height())
```
since the variable is removed maybe `latest processed block` is a better tag for the log.

Reply:
Done in 875004525ff705af02b55818a884567c6e28913a